### DISCUSSION TITLE
View file explorer on remote SSH Jupyterhub

### DISCUSSION THREAD

**Discussion by SamoraHunter** [microsoft/vscode-jupyter#14625] [Questions and Answers]
I cannot view or open the files when connecting to a remote Jupyterhub instance. 

I can ssh to the remote machine in vscode but cannot view the docker container volume for Jupyterhub. 

I would like to see the files in my remote jupyterhub and open notebooks etc in vscode without hosting anything locally.

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/1366
